Xamarin.Forms.Platform.MacOS/Renderers/PickerRenderer.cs

```diff
@@ -77,15 +89,15 @@ protected override void Dispose(bool disposing)
 					}
 
 					if (Control != null)
-						Control.SelectionChanged -= ComboBoxSelectionChanged;
+						Control.Activated -= ComboBoxSelectionChanged;
```

Comment:
Unsubscribe the CollectionChange too here

Reply:
Good find.  I pushed up the changes.  That leads me to some other larger issues at hand.  I am not sure all the work being done in the Dispose method really needs to be happening.  As well as how they are actually being called.  I am seeing minutes worth of time to remove some of our views from scope.  I unfortunately cannot do memory tests right now because the profiler currently cant run on our app.  As soon as we can get that working, I want to run through macOS first and then iOS and verify what is leaking and what isnt and performance test these dispose methods which are again taking minutes to drop one of our views.  I can we we absolutely do have leaks on macOS right now.